### PR TITLE
Alert

### PR DESCRIPTION
**Pull request by vaishnavipaliya11**
Alert
1. primary-alert
2. success-alert
3. information alert
4. danger alert
5. warning alert

### CODE REVIEW

File: alert/alert.html [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+                </div>
+
+                <div class="danger-alert alert-styling">
+                    <p>This is an danger alert.</p>
+                </div>
+
+            </div>
+
+            <div class="div-centered mrg-lrg">
+                <iframe
```
Comment (Tarandeep-s1ngh) [inline alert/alert.html:67]: In iframe, only add code of the component (not of documentation page).

File: alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    padding-left:10px ;
+    margin: 5px;
+    border-radius: 10px;
+}
+
+button{
+    border-radius: 50%;
+    background-color: #808080;
+color:white;
+}
```
Comment (Tarandeep-s1ngh) [inline alert/alert.css]: 1. use class selector instead of element.
2. move this code to CSS of documentation rather than alert component.

**Review by Tarandeep-s1ngh — APPROVED** [review]

File: utility.css [hunk 3]
```diff
@@ -1,51 +1,57 @@
...
 }
 
-.pd-lrg{
+.pd-lrg {
     padding: 2rem;
 }
 
-.mrg-xs{
+.mrg-xs {
     margin: 05.rem;
```
Comment (muskaanshah) [inline utility.css:28]: It will be 0.5rem

**Comment by saketkum11** [comment]
Good to go ahead

**Review by saketkum11 — APPROVED** [review]

File: alert/alert.css [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+    border-color: var(--danger-color);
+}
+
+.alert-styling{
+    padding-left:10px ;
+    margin: 5px;
+    border-radius: 10px;
+}
+
+button{
```
Comment (muskaanshah) [inline alert/alert.css]: Dont use element selector, use classes so that your users can easily incorporate the styles

File: alert/alert.css (hunk 2 above)
Comment (vaishnavipaliya11) [reply alert/alert.css]: its done

**Pull request merged**